recipes/imageio-ffmpeg/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "imageio-ffmpeg" %}
+{% set version = "0.2.0" %}
+{% set sha256 = "2beacb11b121440cfd0bc0caf80da54a3729956002707251ab8e26d3e03933a4" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  script: python -m pip install --no-deps --ignore-installed .
+  number: 0
+
+requirements:
+  build:
+    - python >=3
+    - pip
+  run:
+    - python >=3
+    - ffmpeg
+
+test:
+  imports:
+    - imageio_ffmpeg
+
+about:
+  home: https://github.com/imageio/imageio-ffmpeg
+  doc_url: https://github.com/imageio/imageio-ffmpeg
+  dev_url: https://github.com/imageio/imageio-ffmpeg
+  summary: FFMPEG wrapper for Python
+  license: BSD 2-Clause
+  #license_file: LICENSE (n.a. - should include this in the next release)
```

Comment:
Hi, as long as `upstream` doesn't include the license file in the sdist, could you please add a copy of the license to the recipe directory and include that license here? Thank you!

Reply:
Done

Reply:
Thank you!